ready-to-use fields/README.md

```diff
@@ -0,0 +1,15 @@
+<img align="right" src="https://cdn.miranj.in/mc3/img/apple-touch-icon-precomposed.png" width="57" height="57" alt="Miranj">
+
+# Miranj Craft Boilerplate
+
+A bunch of ready to use fields provided by Miranj to speed-up your project development process. It uses the [Field Manager] plugin (https://github.com/verbb/field-manager) to import the fields.
+
+## Usage
+
+1.  Open the Control Panel and go to _Field Manager_ (see step*1 in \_images* folder).
```

Comment:
The images you’re talking about can directly be referenced in the Markdown file and they will be rendered (using relative URLs). So let us add the images here instead of instructing people to find the relevant one inside a folder.

Reply:
Done.